### PR TITLE
BorgensTidende updates

### PR DESCRIPTION
**Pull request by numismatika**
- convert to current styleguide
- secure subfolders that don't redirect to plain

### CODE REVIEW

**Comment by J0WI** [comment]
This host has a lot of more subdomains. Could you also look for them?

File: src/chrome/content/rules/BergensTidende.xml [hunk 1]
```diff
@@ -55,25 +115,43 @@
...
+	<target host="mediahash2.bt.no" />
+	<target host="mediahash3.bt.no" />
+	<target host="mobil.bt.no" />
+	<target host="multimedia.bt.no" />
+	<target host="nyttig.bt.no" />
+	<target host="old.bt.no" />
+	<target host="reise.bt.no" />
+	<target host="statisk.bt.no" />
+
+	<rule from="^http://statisk\.bt\.no/" to="https://statisk.bt.no/" />
```
Comment (J0WI) [inline src/chrome/content/rules/BergensTidende.xml]: This is already covered by the generic redirect rule below.

File: src/chrome/content/rules/BergensTidende.xml [hunk 2]
```diff
@@ -55,25 +115,43 @@
...
+
+	<rule from="^http://www\.bt\.no/(external/|public/|resources/|skins/)" to="https://www.bt.no/$1" />
```
Comment (J0WI) [inline src/chrome/content/rules/BergensTidende.xml]: Everything else will also be redirected by the generic redirect rule below.
Why do you use this redirect here? Also the reason for the exclusion on `/$` is not clear to me.

Comment (J0WI) [inline src/chrome/content/rules/BergensTidende.xml]: Ah, but an exclusion for `http://www.bt.no/tv/` is required.

Comment (numismatika) [inline src/chrome/content/rules/BergensTidende.xml]: @J0WI if the top folder redirects to plain but subfolders can be secured, you need an exclusion to make the test script happy.

Comment (J0WI) [inline src/chrome/content/rules/BergensTidende.xml]: I was not able to reproduce that. Can you confirm?

Comment (numismatika) [inline src/chrome/content/rules/BergensTidende.xml]: I will revisit all these norwegian news pages when i got time.
Might become a lot simpler if i extrapolate correctly from the aftenposten website.

**Comment by J0WI** [comment]
This does not pass our tests.

**Comment by numismatika** [comment]
@J0WI now they pass

File: src/chrome/content/rules/BergensTidende.xml [hunk 3]
```diff
@@ -1,79 +1,181 @@
 <!--
 	Bergens Tidende
 
-	For rules causing false/broken MCB, see BT.no-falsemixed.xml.
```
Comment (J0WI) [inline src/chrome/content/rules/BergensTidende.xml]: See also `BT.no-falsemixed.xml`

File: src/chrome/content/rules/BergensTidende.xml [hunk 4]
```diff
@@ -1,79 +1,181 @@
...
-
+	Cert chain issue:
+		- scenario.bt.no
+
+	Cert expired:
+		- nyhetsbrev.bt.no
+		- pust.bt.no
+
+	Cert mismatch:
+	- a.bt.no
```
Comment (J0WI) [inline src/chrome/content/rules/BergensTidende.xml]: Please use a consistent amount of tabs to indent the comments.

File: src/chrome/content/rules/BergensTidende.xml (hunk 3 above)
Comment (numismatika) [inline src/chrome/content/rules/BergensTidende.xml]: removed

**Comment by J0WI** [comment]
Thanks!

**Pull request merged**